### ISSUE TITLE
Fix `selector-pseudo-class-no-unknown` false positives for `:active-view-transition` and `:active-view-transition-type()`

### ISSUE THREAD

**Issue by Mouvedia** [stylelint/stylelint#7861]
### What minimal example or steps are needed to reproduce the bug?

```css
:active-view-transition {}
:active-view-transition-type(foo) {}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "selector-pseudo-class-no-unknown": true
  }
}
```

### How did you run Stylelint?

https://stylelint.io/demo

### References

https://caniuse.com/mdn-css_selectors_active-view-transition-type
https://caniuse.com/mdn-css_selectors_active-view-transition

**Comment by Mouvedia** [comment]
I've labeled the issue as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.

There are [steps on how to fix a bug in a rule](https://stylelint.io/developer-guide/rules#fix-a-bug-in-a-rule) in the Developer guide.

**Comment by romainmenke** [comment]
Demo : https://stylelint.io/demo/#N4Igxg9gJgpiBcCCGYAuBLAbjAtJ9MA7jqgE5IB2AzuhhBQATAOQA2Ep8DpMUA3AwC+AHQrwUGbHgLEylGnQokAngAcYACgBmECAEomLCO07deAwSAA0ILelYwAckgC2cRDAAer1Q4B0YFRU1uD0dgDmCCDAogwMwiBeqDAUUFQJXADasXHxIFSoyg6s6BSoOJAUETgFlFBIpFAJOQC6opY2lREAYhwuSKhRAFZU9CGwqsGIMYx5BUUwJWUZeawDMAUJVjkJ88Wl5V3o4TWodQ1NCKvrmyDtIIJAA
